pyproject.toml

```diff
@@ -0,0 +1,57 @@
+[build-system]
+requires = ["setuptools>=68.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+platforms = ["Linux x86, x86-64"]
+
+[tool.setuptools.packages.find]
+include = ["sopel_slap", "sopel_slap.*"]
+namespaces = false
+
+[project]
+name = "sopel-slap"
+version = "0.2.0.dev0"
+description = "Plugin for Sopel that lets users slap each other in fun ways"
+keywords = [
+  "sopel",
+  "plugin",
+  "slap",
+  "pranks",
+  "bot",
+  "irc",
+]
+maintainers = [
+  { name="dgw", email="dgw@technobabbl.es" },
+]
+authors = [
+  { name="dgw", email="dgw@technobabbl.es" },
+  { name="Florian Strzelecki", email="florian.strzelecki@gmail.com" },
```

Comment:
Is this left over from a copy-paste? As far as I know, you have not written any code for this plugin yet. 😝

I think it would make the most sense to credit the original author as acknowledged in the README and plugin file header—probably just as "Michael Yanovich", with no email address, so no one bothers them.

_Maybe_ I've done enough [in the past](https://github.com/sopel-irc/sopel-extras/commits/master/slap.py) to be considered a co-author, too. But I also wouldn't mind being only a maintainer.

Reply:
Clearly a copy/paste. Next PR, I'll add my name. 😄 

I think you've done enough regarding copyright.